### ISSUE TITLE
Wrong current version in README.md

### ISSUE THREAD

**Issue by ryandesign** [jrincayc/ucblogo-code#127]
README.me says

> The current version is 6.2

but the current version as of 4 months ago is actually 6.2.2. Maybe it would be simpler not to list the current version in the readme so that you don't have to keep updating it.

**Comment by jrincayc** [comment]
Yes, I have not been updating the version in the readme for point releases (6.2.1, 6.2.2 ...)

**Comment by jrincayc** [comment]
(I will fix this next release, or happily accept a pull request that remove this from the readme or fixes this.)
